docs/usage.md

```diff
@@ -92,28 +98,28 @@ Magmap can handle several types of input that can be used for different purpose.
 The indexes input is used by Sourmash to select genomes that can be downloaded in a second step and added to the pipeline.
 It is provided with the `--indexes` parameter and it is a path (local or remote).
 
-```
-https://github.com/nf-core/test-datasets/raw/magmap/testdata/sourmash_test.index.sbt.zip
+```bash
+nextflow run nf-core/magmap -profile docker --outdir results/ --input samples.csv --genomeinfo localgenomes.csv --indexes 'https://github.com/nf-core/test-datasets/raw/magmap/testdata/sourmash_test.index.sbt.zip'
 ```
 
 N.B.: the sbt files can be generated with sourmash (check it [here](https://sourmash.readthedocs.io/en/latest/index.html)) or some can be found in sourmash documentation website.
 E.g. "sourmash gtdb sbt" you will find the indexes for gtdb.
 
 #### Metadata input
 
-Magmap accepts several metadata as `.csv` files that provides information about the genomes that you will use in the pipeline.
+Magmap accepts several metadata as `.csv` files that provides information about the genomes that you will use in the pipeline. Each metadata can be used to get different information about you genomes as their quality (CheckM) or the taxonomy (gtdb). Magmap will merge all these tables and create a new one: each row will correspond to a genome (based on its accno) followed by several columns.
 
 ##### gtdb_metadata
 
-This file contains several information that can be found in gtdb metadata on the official [website](https://data.ace.uq.edu.au/public/gtdb/data/releases/release80/80.0/). You can either use directly the same file found there or you can make a custom one.
+This file contains several information that can be found in gtdb metadata on the official [website](https://data.ace.uq.edu.au/public/gtdb/data/releases/release220/220.0/bac120_metadata_r220.tsv.gz). You can either use directly the same file found there or you can make a custom one.
 
 ##### gtdb-tk metadata
 
-This file contains several information that can be obtained as gtdb-tk output. You can either use directly the same file found there or you can make a custom one.
+This file contains several information that can be obtained as gtdb-tk output, [see](https://ecogenomics.github.io/GTDBTk/files/markers_summary.tsv.html). You can either use directly the output file or you can make a custom one.
 
 ##### checkM metadata
 
-This file contains several information that can be obtained as checkM output. You can either use directly the same file found there or you can make a custom one.
+This file contains several information that can be obtained as checkM output, [see](https://github.com/nf-core/test-datasets/blob/magmap/testdata/checkm2.quality_report.tsv). You can either use directly the output file found there or you can make a custom one.
```

Comment:
```suggestion
This file should be formatted like the checkM output, [see](https://github.com/nf-core/test-datasets/blob/magmap/testdata/checkm2.quality_report.tsv).
```

Reply:
You link to the wrong file: CheckM2, but this is the CheckM section.

Reply:
I changed the name